### ISSUE TITLE
Add support for atcoder

### ISSUE THREAD

**Issue by cdbalaji** [CodeDrills-CD/codedrills-recommender#11]
Add atcoder support. Suggested prefix: `at:`

**Comment by ankur-kayal** [comment]
Research work done:

1. Atcoder has no APIs to fetch problems but we can scrape all the problem data ( sample implementation: https://github.com/kenkoooo/AtCoderProblems/tree/master/atcoder-problems-backend). The only caveat is that atcoder problems don't have problem tags which is the basis of the codedrills recommender).
2. Incorrect solution to the above problem (although it might work till some extent). Atcoder has official solution to the problems, so we can scrape the solution title and scrape the solution heading like bfs approach would give the tag bfs. Might not work for most cases.

**Comment by ashishpawar517** [comment]
Is there way we can fetch the user submissions Atleast?

**Comment by ankur-kayal** [comment]
@ashishpawar517 yes we can get user submissions but how will that help? We will not be able to recommend problems from atcoder to the user.

**Comment by ashishpawar517** [comment]
Actually there's one website called as atcoder tags that has some number of problems with tags, we use that for recommendations. Actually thing that I wanted to see was how can we add other cp sites submissions in codedrills like getting entire cp tracking from all the websites to one platform

**Comment by ashishpawar517** [comment]
any updates to this @ankur-kayal
